src/Microsoft.OData.Client/Serialization/HttpClientRequestMessage.cs

```diff
@@ -89,6 +91,24 @@ public HttpClientRequestMessage(DataServiceClientRequestMessageArgs args)
             }
         }
 
+#if NETSTANDARD2_0_OR_GREATER
+        /// <summary>
+        /// Gets the underlying <see cref="HttpClientHandler.ServerCertificateCustomValidationCallback"/> so that callers can customize their certificate valiation
+        /// during OData requests
+        /// </summary>
+        public Func<HttpRequestMessage, X509Certificate2, X509Chain, SslPolicyErrors, bool> ServerCertificateCustomValidationCallback
+        {
+            get
+            {
+                return this._handler.ServerCertificateCustomValidationCallback;
+            }
+            set
+            {
+                this._handler.ServerCertificateCustomValidationCallback = value;

```

Comment:
shall we check the input 'null'?

Reply:
I assume null is valid to clear the callback. Regardless, by passing it down we can simply reflect any validation that the HttpClientHandler may already be imposing, rather than masking lower level behavior/validation.